zilliqa/src/consensus.rs

```diff
@@ -722,6 +729,10 @@ impl Consensus {
         if let Some(priority_txs) = self.new_transactions_priority.get_mut(&removed.signer) {
             priority_txs.retain(|tx| tx.hash != *tx_hash);
         }
+
+        // Finally, insert tx into the db, so there is no discontinuity where the TX can't
+        // be found anywhere
+        let _ = self.db.insert_transaction(tx_hash, &removed.tx);
```

Comment:
This discards the Result, right - any particular reason rather than bubbling it up?

Reply:
My auto formatter inserted it! I will fix in subsequent, thanks